packages/alfa-aria/src/features/h1.ts

```diff
@@ -0,0 +1,20 @@
+import { Element, getAttribute, Node } from "@siteimprove/alfa-dom";
+import { Feature, Role } from "../types";
+import * as Roles from "../roles";
+
+/**
+ * @see https://www.w3.org/TR/html-aria/#h1-h6
+ */
+export const H1: Feature = {
+  element: "h1",
+  role,
+  allowedRoles: [Roles.Tab, Roles.None, Roles.Presentation]
+};
+
+function role(h1: Element, context: Node): Role | undefined {
+  const att = getAttribute(h1, "aria-level");
+  if (att !== null && parseInt(att) > 0) {
+    return Roles.Heading;
+  }
+  return undefined;
+}
```

Comment:
The spec is slightly confusing on the point of headers. The headers will get an implicit `aria-level` of a positive integer matching the header level, e.g. an `<h1>` would get an implicit `aria-level="1"`. The `role` is therefore always `heading`, though we will need to find a way of encoding the implicitly added ARIA attributes for these cases.

Reply:
Let's discuss this